invenio_accounts/cli.py

```diff
@@ -61,6 +63,8 @@ def users_create(email, password, active, confirm):
         kwargs["active"] = active
         if confirm:
             kwargs["confirmed_at"] = datetime.utcnow()
+        if profile:
+            kwargs["user_profile"] = json.loads(profile)
```

Comment:
Unfortunately, `invenio-accounts` does not depend on `invenio-userprofiles`, but the other way around. So, it should not be possible (ideally) to do this, in particular if the `user_profile` schema will change.
Was this discussed?

Reply:
[`user_profile` is not part of the accounts model](https://github.com/inveniosoftware/invenio-accounts/blob/e2759b5a56181df27ed750f1e336fad84e0752b4/invenio_accounts/models.py#L119-L125). So as long as the passed JSON conforms to the configurable schema we're validating the input properly.

Reply:
> [`user_profile` is part of the accounts model](https://github.com/inveniosoftware/invenio-accounts/blob/e2759b5a56181df27ed750f1e336fad84e0752b4/invenio_accounts/models.py#L119-L125). So as long as the passed JSON conforms to the configurable schema we're validating the input properly.

Oh, thanks for reminding me that, I was convinced it was not, didn't even check :) Sorry about that!